client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout.js

```diff
@@ -35,17 +36,27 @@ import CartFreeUserPlanUpsell from 'my-sites/checkout/cart/cart-free-user-plan-u
 import MaterialIcon from 'components/material-icon';
 import Gridicon from 'components/gridicon';
 
+const debug = debugFactory( 'calypso:wp-checkout' );
+
 const ContactFormTitle = () => {
 	const translate = useTranslate();
 	const isActive = useIsStepActive();
 	const isComplete = useIsStepComplete();
 	const [ items ] = useLineItems();
+	const isGSuiteInCart = items.find(
+		( item ) => !! item.wpcom_meta?.extra?.google_apps_users?.length
+	);
 
 	if ( areDomainsInLineItems( items ) ) {
 		return ! isActive && isComplete
 			? translate( 'Contact information' )
 			: translate( 'Enter your contact information' );
 	}
+	if ( isGSuiteInCart ) {
+		return ! isActive && isComplete
+			? translate( 'G Suite account information' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'G Suite Account Information' )` *ES Score: 9*